app/models/user.rb

```diff
@@ -10,4 +11,14 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true,
     length: {minimum: Settings.minium_password}
+
+  # Returns the hash digest of the given string.
+  def self.digest string
```

Comment:
` self` -> thèn này ở đây nghĩa là gì chú

method `digest` dùng để làm gì luôn

Reply:
self dùng để định nghĩa class method.
method digest dùng để hash string ban đầu theo thuật toán bcrypt.

> ` self` -> thèn này ở đây nghĩa là gì chú
> 
> method `digest` dùng để làm gì luôn

dạ ^^
self dùng để định nghĩa class method.
method digest dùng để hash string ban đầu theo thuật toán bcrypt.